src/ApplicationInsights.ServiceFabric.Native/Net45/IWrappingClient.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Microsoft.ApplicationInsights.ServiceFabric.Remoting.Activities
+{
+    using Microsoft.ServiceFabric.Services.Remoting.Client;
+
+    /// <summary>
+    /// Interface implemented by remoting clients to provide access to the inner client.
+    /// </summary>
+    internal interface IWrappingClient
```

Comment:
what is the purpose of this interface? why not use IServiceRemotingClient directly?